packages/sql-mapper/test/entity_transaction.test.js

```diff
@@ -42,7 +42,6 @@ test('entity transactions', async ({ equal, same, teardown, rejects }) => {
     same(findResult, [{ title: 'foo' }, { title: 'bar' }])
 
     try {
-      console.log('isSQLite', mapper.db.isSQLite)
```

Comment:
We should have the linter catch these, IMO...

Reply:
I don't think it's really possible.